pyvista/core/filters/data_set.py

```diff
@@ -956,43 +957,54 @@ def threshold(
            thresholding depends on whether that point is part of a cell that
            is kept after thresholding.
 
+           Please also note the default ``preference`` choice for CELL data
+           over POINT data. This is contrary to most other places in PyVista's
+           API where the preference typically defaults to POINT data. We chose
+           to prefer CELL data here so that if thresholding by a named array
+           that exists for both the POINT and CELL data, this filter will
+           default to the CELL data array while performing the CELL-wise
+           operation.
+
         Parameters
         ----------
         value : float or sequence, optional
-            Single value or (min, max) to be used for the data threshold.  If
+            Single value or (min, max) to be used for the data threshold. If
```

Comment:
I'm not sure what happens if the user passes a sequence in order (max, min).  I don't think this is explicitly handled here, and I'm not sure what vtk does with that (I assume it returns no cells?) Should we throw a warning?

Reply:
Hm. This will result in no-output. We should likely raise an Error rather than a warning. ParaView doesn't allow the user to do this

Reply:
Added a `ValueError` for this